va/va.h

```diff
@@ -959,6 +974,25 @@ typedef union _VAConfigAttribValEncJPEG {
 /** \brief Driver supports intra refresh of multiple reference encoder */
 #define VA_ENC_INTRA_REFRESH_MULTI_REF                  0x00040000
 
+/**@}*/
+/** @name Attribute values for VAConfigAttribHistogram. */
+/**@{*/
+/** \brief Specifying histogram channels */
+#define VA_HISTOGRAM_NONE                             0x00000000
+/** \brief Driver supports channel Y histogram report  */
+#define VA_HISTOGRAM_CHANNEL_Y                                0x00000001
+/** \brief Driver supports channel U histogram report  */
+#define VA_HISTOGRAM_CHANNEL_U                                0x00000002
+/** \brief Driver supports channel V histogram report  */
+#define VA_HISTOGRAM_CHANNEL_V                                0x00000004
+/** \brief Driver supports channel R histogram report  */
+#define VA_HISTOGRAM_CHANNEL_R                                0x00000008
```

Comment:
for R, G, B, do we need to give distinct values for Y, U, V?

Reply:
this is for the caps reporting, we can t report 0x7 only, customer dont know whether we support  RGB or YUV from value 0x7